### ISSUE TITLE
Very low resolution icon on Linux Mint

### ISSUE THREAD

**Issue by GuidoBartoli** [wesnoth/wesnoth#3312]
Hi,
I noticed that when switching among active apps using ALT+TAB, the Wesnoth icon is at a _really low_ resolution compared to the other ones.

Here is a screenshot to compare it against the Google Chrome icon (please view it at 100% to see the issue). I'm using Wesnoth 1.14.3 installed with flatpak on Linux Mint 18.3 Cinnamon 64-bit.

![wesnoth](https://user-images.githubusercontent.com/16103676/42231394-bafe84b8-7eeb-11e8-991b-d438534e881f.jpg)

**Comment by sevu** [comment]
@loonycyborg wondering if this is flatpak related... I didn't find a way to download the archive from flathub, can you check if the icons are inside. Maybe also the naming scheme is not the same in flathub with the regards to the org.wesnoth.Wesnoth scheme used elsewhere.

**Comment by loonycyborg** [comment]
Wesnoth 1.14.7 is available on flathub atm, there were some changes that might affect icons since then. I looked inside 1.14.7 and it seems to include icons of sizes starting from 16x16 up to 512x512. Mind you there still could be bugs that could cause particular DE to use wrong icon

**Comment by GuidoBartoli** [comment]
Ok, thanks for the reply, I will check it out this evening and report my findings.

**Comment by GuidoBartoli** [comment]
I checked with Wesnoth 1.14.7 installed with flatpak on Linux Mint 19.1 (based on Ubuntu 18.04) and the low resolution icon when switching apps is still present, so the bug is not fixed yet.
To be honest, with this version I noticed also that the mouse cursor shape is pretty bad when "show color cursor" option is enabled in Advanced. It is solid-colored in brown, without any ornament like the one with the classical PPA version. I cannot attach screenshots, because I could not make the screen grabbing utility include the mouse cursor... :(

**Comment by loonycyborg** [comment]
Which exact environment you're using? Gnome? Cinnamon? Cursor issue is known already and seems to affect only if ran via x11 compatibility instead of wayland.

**Comment by GuidoBartoli** [comment]
I'm using Linux Mint 19.1 with Cinnamon 4.0.10 (light theme)

**Comment by loonycyborg** [comment]
I tried it on fedora/gnome and alt-tab icon was fine. Do you have hidpi or something like that?

**Comment by GuidoBartoli** [comment]
Mmmh no, just a standard Samsung 23" at 1920x1080p.

**Comment by Wedge009** [comment]
I had a look around on Windows 7, Windows 10, and KDE, and I feel this may be more a question of how Guido's desktop environment handles the icon information. I did notice the icon in the Windows 7 task bar to be a little grainy compared with other applications but not to the level that I see in the screenshot. And the Wesnoth icon supports a maximum 256x256 so it's not as though the icon information isn't there.

In other words, I'm not sure this is something that can be resolved within the scope of this project.

**Comment by GregoryLundberg** [comment]
This sounds more like a downstream issue for the Mint package maintainer to address.

**Comment by Wedge009** [comment]
That was my thought. Still need to keep this?

**Comment by GuidoBartoli** [comment]
> This sounds more like a downstream issue for the Mint package maintainer to address.

I think that too... if everything has been checked on Wesnoth side, for me we can close this issue.

**Comment by Wedge009** [comment]
I think we've done what we can to assist. Thanks for your acknowledgement.
